torch_geometric/nn/conv/rgcn_conv.py

```diff
@@ -220,6 +221,8 @@ def forward(self, x: Union[OptTensor, Tuple[OptTensor, Tensor]],
                 out = out + h.contiguous().view(-1, self.out_channels)
 
         else:  # No regularization/Basis-decomposition ========================
+            if is_compiling():
+                disable_use_segment_matmul()
```

Comment:
Sorry for asking so many questions. What was the reason for the graph break in the first place? Computing the heuristic? Can't we just do
```
if is_compiling():
    use_segment_matmul = False
else:
    use_segment_matmul = torch_geometric.backend.use_segment_matmul
    if use_segment_matmul is None:
        pass
```
etc